packages/core/src/sequence.ts

```diff
@@ -46,6 +47,7 @@ export class Sequence {
     @inject('findRoute') protected findRoute: FindRoute,
     @inject('invokeMethod') protected invoke: InvokeMethod,
     @inject('logError') protected logError: LogError,
+    @inject('authenticate') protected authenticate: AuthenticateFn,
```

Comment:
I think the default sequence defined by `@loopback/core` should not include a call to `authenticate`, because that feature is not included in `@loopback/core`.

If we decide to call it anyways, then `@loopback/core` should provide a default dummy implementation that does nothing.